DTML.EduBot/Constants/BotEntities.cs

```diff
@@ -14,5 +14,44 @@ public static class BotEntities
         public static string Color = "color";
         public static string Subject = "subject";
         public static string encyclopediaOrganization = "builtin.encyclopedia.organization.organization";
+        public static string TranslateTargetLanguage = "Translate.TargetLanguage";
```

Comment:
Can you change the type of all these strings from static to const?

it will look like `public const string TranslateTargetLanguage = "Translate.TargetLanguage";`